orchestrator/src/main/kotlin/Orchestrator.kt

```diff
@@ -472,12 +474,12 @@ class Orchestrator(
 typealias CreatedJobs = List<JobScheduleFunc>
 
 /**
- * Create an [Issue] object representing an error that occurred in any [Endpoint].
+ * Create an [Issue] representing the error with the given [message], if any, that occurred in an [Endpoint].
  */
-fun <T : Any> Endpoint<T>.createErrorIssue(): Issue = Issue(
+fun <T : Any> Endpoint<T>.createErrorIssue(message: String?): Issue = Issue(
```

Comment:
Nit: Should this function expect the whole `WorkerMessage` object instead of the message string and handle the extraction of the message itself? Then it would be fully responsible for the issue creation, and `handleWorkerError` would not need to partially deal with this.

Reply:
I basically like the idea. However, as `handleWorkerError` takes a `WorkerMessage`, and that interface has nothing in common with `WorkerErrorMessage`, this requires `createErrorIssue` to take `error: Any`. But that's probably acceptable as the code still looks cleaner, IMO.